projects/laji/src/app/+project-form/about/about.component.html

```diff
@@ -1,25 +1,28 @@
 <laji-technical-news></laji-technical-news>
 
-<ng-container *ngIf="aboutData$ | async as data">
+<div *ngIf="aboutData$ | async as data" lajiFormOption="mobile dataset about collectionID options.disableRequestDescription">
```

Comment:
It looks to me that `mobile dataset about` should be `options.mobile options.dataset options.about` since they are under the options object

Reply:
Because these are just strings and the types are not used, it's easy to make mistakes. It would be better if it would use types but I don't know if it would take too much work at this point. It could look something like this:

This file:
```
<div *ngIf="aboutData$ | async as data" [lajiFormOption]="['collectionID']" [lajiFormOptionOption]="['mobile', 'dataset' , 'disableRequestDescription']">
```

Directive:
```
selector: '[lajiFormOption], [lajiFormOptionOption]'
...
@Input() lajiFormOption: (keyof SchemaForm)[];
@Input() lajiFormOptionOption: (keyof FormOptions)[];
ngOnChanges() { // do something with the options }
```

Reply:
It's a nice idea, but with this approach we'd have to have a different directive for each depth level, which would be messy. For example named place options are in an object inside the form options. Currently those set simply with e.g. `options.namedPlaceOptions.useLabel`.

It's easy to make mistakes yes consequences of a typo aren't that dangerous. The syntax is also less bloaty in the current way it's done.